uvicorn/protocols/http/h11_impl.py

```diff
@@ -48,7 +48,13 @@ def __init__(
         self.logger = logging.getLogger("uvicorn.error")
         self.access_logger = logging.getLogger("uvicorn.access")
         self.access_log = self.access_logger.hasHandlers()
-        self.conn = h11.Connection(h11.SERVER)
+        if config.h11_max_incomplete_event_size:
+            self.conn = h11.Connection(
+                h11.SERVER,
+                max_incomplete_event_size=config.h11_max_incomplete_event_size,
+            )
+        else:
+            self.conn = h11.Connection(h11.SERVER)
```

Comment:
Maybe we should set the default ourselves instead of using the one h11 defines? Even if it's with the same value?

Just a thought.

Reply:
I think it's better to respect the h11 default in case they change it for some reason.

Reply:
I think the reason you mentioned is exactly why we should set it here :sweat_smile:

Maybe we should check what hypercorn is doing here :thinking:

Reply:
> If they change it, they probably have a good reason for doing so, like fixing a bug or implementing a new feature.

I think we need to be in charge here. `h11` can change their values as they want, they are on pre-stable release (I doubt they would, because this value is actually based on several other implementations).

Also, the code becomes cleaner anyway.

Would you mind changing it?